configs/fedora-sig-i686-steam.yaml

```diff
@@ -0,0 +1,46 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Packages Required By Steam Spec File
+  description: Packages Required By Steam
+  maintainer: ssmoogen
+  packages:
+    # The minimum development environment is this:
+    - desktop-file-utils
+    - libappstream-glib
+    - make
+    - systemd
+    - tar
+    - zenity
+    # Requirements from https://github.com/rpmfusion/steam/blob/master/steam.spec
+    - NetworkManager-libnm
```

Comment:
All we would need to consider here are those requires which are `%{?_isa}`-suffixed, and therefore they could be listed here as `NetworkManager-libnm.i686` and so on.